snmp-notifier/values.yaml

```
@@ -0,0 +1,80 @@
+---
+# Default values for dellemc-supportassist-ese
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# These global settings relate to supportassist-ese
+global:
+  # Name of a Kubernetes secret with Docker credentials for the private registry
+  registrySecret: dockerhub-registry-secret
+
+  # The registry hosting containers for Dell based k8s products
+  registry: emccorp
+
+# The default docker tag and pull policy for dell-supportassist-ese
+tag: 2.76.0
+pullPolicy: Always
+
+# enabled - true - enable snmp-notifier false - disable it
+enabled: true
+
+# productName: objectscale - required field to pass in using --set
+
+# The image configured for snmp notifier 
+image:
+  repository: snmp-notifier
+  # tag: # rfw-update-this snmp-notifier-image
```

```suggestion
  tag: # rfw-update-this snmp-notifier-image
```
remove comment so that its auto-updated by release framework.